types/rbac_test.go

```diff
@@ -47,7 +47,7 @@ func TestRuleValidate(t *testing.T) {
 	assert.NoError(t, r.Validate())
 
 	// Wildcard org
-	r.Organization = "*"
+	r.Organization = types.OrganizationTypeAll
```

Comment:
@bitmonk Ha, looks like we also missed that one. The tests are actually complaining about that: https://travis-ci.org/sensu/sensu-go/jobs/401827784#L584

Basically, you should do `r.Organization = OrganizationTypeAll` since it's within the same package, so no need to explicitly refer to it.

Reply:
gotcha! thanks. :)

looks like i was only running e2e, and i should also run `./build.sh lint` ?

Reply:
@bitmonk You can do `./build.sh quality` to run the linter and unit tests, and `./build.sh integration` for integration tests. That should run the same suite of tests that Travis is failing on.

Reply:
Otherwise you could just run `./build.sh`, without any argument, if you want to keep it simple; it will run every possible tests and also install all dependencies and tools for the tests.